Ical.Net.Tests/MatchTimeZoneTests.cs

```diff
@@ -80,21 +80,25 @@ public void MatchTimeZone_LocalTimeJapanWithTimeZone()
         var evt = calendar.Events.First();
         var until = evt.RecurrenceRules.First().Until;
 
-        var expectedUntil = new DateTime(2023, 11, 05, 13, 00, 00, DateTimeKind.Utc);
-        var occurrences = evt.GetOccurrences(new CalDateTime(2023, 11, 01), new CalDateTime(2023, 11, 06));
+        var expectedUntil = new DateTime(2024, 10, 05, 14, 0, 0, DateTimeKind.Utc);
+        var occurrences = evt.GetOccurrences(new CalDateTime(2024, 10, 01), new CalDateTime(2024, 10, 07));
 
         Assert.Multiple(() =>
         {
             Assert.That(until, Is.EqualTo(expectedUntil));
             Assert.That(occurrences.Count, Is.EqualTo(5));
             /*
                Should have 5 occurrences:
-               November 1, 2023: 09:00 AM - 10:00 AM (UTC+0900) (Asia/Tokyo)
-               November 2, 2023: 09:00 AM - 10:00 AM (UTC+0900) (Asia/Tokyo)
-               November 3, 2023: 09:00 AM - 10:00 AM (UTC+0900) (Asia/Tokyo)
-               November 4, 2023: 09:00 AM - 10:00 AM (UTC+0900) (Asia/Tokyo)
-               November 5, 2023: 09:00 AM - 10:00 AM (UTC+0900) (Asia/Tokyo)
-             */
+               October 1, 2024: 01:00 AM - 02:00 AM (UTC+1000) (Australia/Sydney)
+               October 2, 2024: 01:00 AM - 02:00 AM (UTC+1000) (Australia/Sydney)
+               October 3, 2024: 01:00 AM - 02:00 AM (UTC+1000) (Australia/Sydney)
+               October 4, 2024: 01:00 AM - 02:00 AM (UTC+1000) (Australia/Sydney)
+               October 5, 2024: 01:00 AM - 02:00 AM (UTC+1000) (Australia/Sydney)
+
+               October 6, 2024: 01:00 AM - 02:00 AM (UTC+1100) (Australia/Sydney)
```

Comment:
Oh, actually I think, this *should* be included, because `UNTIL` must be considered *inclusive*.

> If the value
      specified by UNTIL is synchronized with the specified recurrence,
      this DATE or DATE-TIME becomes the last instance of the
      recurrence.

Reply:
Yes, the conclusion came from the description I assume. It was not updated.
Now there are 2 test cases, one when DST occurs.

Reply:
Yes, I was looking at the comments. But now reconsidering the test, I still think its not correct.

October 6, 2024: 01:00 AM Sydney time = 20241005T140000Z, so it should be included in the first test and the number of occurrences should therefore be 6 in both cases.

Reply:
Isn't UTC Time: 2024-10-05 14:00:00 => Sydney Time (AEST, UTC+10:00): 2024-10-06 00:00:00?

Reply:
Oh yes, of course, sorry for the confusion!